package.json

```diff
@@ -48,6 +48,8 @@
     "http-status": "^1.7.4",
     "immutability-helper": "^3.1.1",
     "next": "^14.1.4",
+    "nextjs-progressloader": "^1.2.0",
+    "nprogress": "^0.2.0",
```

Comment:
Using `NProgress` as a direct dependency here seems unnecessary since it's only a peer dependency of `nextjs-progressloader`

Reply:
removes in the commit below